src/pages/ResetPassword.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { resetPassword } from '../services/member';
+
+const ResetPasswordForm: React.FC = () => {
+  const [newPassword, setNewPassword] = useState('');
+  const [confirmPassword, setConfirmPassword] = useState('');
+  const [error, setError] = useState('');
```

Comment:
would it be more consistent to set these as undefined initially?

Reply:
(not all just error and success)

Reply:
Yeah, I'm going to redo this page later. This is a shitty version of it that chatgpt made